instaclustr/resource_vpc_peering.go

```diff
@@ -48,8 +48,18 @@ func resourceVpcPeering() *schema.Resource {
 			},
 
 			"peer_subnet": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"peer_subnets"},
+			},
+
+			"peer_subnets": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:      true,
+				ConflictsWith: []string{"peer_subnet"},
```

Comment:
asking for my reference. does "conflictswith" reject requests which contain both?